CONFIGDOC.md

```diff
@@ -97,6 +108,210 @@ You can define several axes in the **Configuration Matrix** section. For example
+
+
+```groovy
+// Declarative Pipeline
+pipeline {
+    agent any
+    stages{
+        stage('Run MATLAB Tests') {
+            steps
+            {
+                runMATLABTests(testResultsJUnit: 'test-results/results.xml',
+                               codeCoverageCobertura: 'code-coverage/coverage.xml')
+            }       
+        }                
+    } 
+}
+``` 
+
+Define a Scripted Pipeline to run your tests and generate artifacts.
+
+```groovy
+// Scripted Pipeline
+node {
+    runMATLABTests(testResultsJUnit: 'test-results/results.xml',
+                   codeCoverageCobertura: 'code-coverage/coverage.xml') 
+}
+``` 
+
+
+ 
+ This table lists the name-value pair arguments that you can specify with the `runMATLABTests` step.
+
+| Name               	| Value                                                                                                                	|
+|-------------------------	|----------------------------------------------------------------------------------------------------------------------------	|
+| testResultsPDF          	| Path to write test results report in PDF format (currently not supported on Mac platforms).<br/>**Example:** `'test-results/results.pdf'`                  	|
+| testResultsTAP          	| Path to write test results report in TAP format.<br/>**Example:** `'test-results/results.tap'`                  	|
+| testResultsJUnit        	| Path to write test results report in JUnit XML format.<br/>**Example:** `'test-results/results.xml'`            	|
+| testResultsSimulinkTest 	| Path to export Simulink Test&trade; Manager results in MLDATX format (requires Simulink Test license and is supported in MATLAB R2019a or later).<br/>**Example:** `'test-results/results.mldatx'` 	|
+| codeCoverageCobertura   	| Path to write code coverage report in Cobertura XML format.<br/>**Example:** `'code-coverage/coverage.xml'`     	|
+| modelCoverageCobertura  	| Path to write model coverage report in Cobertura XML format (requires Simulink Coverage™ license and is supported in MATLAB R2018b or later).<br/>**Example:** `'model-coverage/coverage.xml'`   	|
+
+
+## Use MATLAB in Matrix Build
+Similar to multi-configuration projects, you can use MATLAB as part of a [matrix](https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-matrix) build in Pipeline projects. For example, you can define a Declarative Pipeline to run your test suite on different platforms or against different versions of MATLAB. (Matrix build is not supported in Scripted Pipelines.)
+
+Your Pipeline must have a `matrix` section to define the possible name-value combinations that should run in parallel. This example shows how to define a Pipeline to run your MATLAB code and generate test artifacts using MATLAB R2018b, R2019a, and R2020a.

```

Comment:
The example is showing R2018b, 19a, and 19b. Not 20a.

Reply:
Fixed.